WORKSPACE

```diff
@@ -202,6 +202,28 @@ npm_import(
     version = "8.4.0",
 )
 
+############################################
+# Stardoc
+load("@io_bazel_stardoc//:setup.bzl", "stardoc_repositories")
+
+stardoc_repositories()
+
+load("@rules_jvm_external//:repositories.bzl", "rules_jvm_external_deps")
+
```

Comment:
Why are all the jvm things required now?

Reply:
Stardoc 0.6.2 has really bad transitive deps when using WORKSPACE 🤷